src/lib/track-changes-utils.ts

```diff
@@ -110,3 +115,102 @@ export function getActualAttrs<T extends ManuscriptNode>(node: T) {
   }
   return attrs
 }
+
+export const getAttrsTrackingButton = (changeID: string) => {
+  const el = document.createElement('button')
+  el.className = 'attrs-popper-button'
+  el.value = changeID
+  el.innerHTML = editIcon
+
+  return el
+}
+
+const editIcon = `
+ <svg
```

Comment:
Can we move it to styleguide?

Reply:
if we move it to style-guide it should be a react component to be useful for other components in the app, and I would rather to feed prosemirror view a string rather than rendering a react component.